src/Kestrel.Core/Internal/Http/HttpMethod.cs

```diff
@@ -16,5 +16,7 @@ public enum HttpMethod: byte
         Options,
 
         Custom,
+
+        Unknown = byte.MaxValue
```

Comment:
It's best practice to make Unknown/None the zero value, the default for uninitialized fields.

Reply:
Its used as the [index into `_methodNames` array](https://github.com/aspnet/KestrelHttpServer/blob/5bf7d3b28a2fcca56a7d24fd7e79c13dedae657b/src/Kestrel.Core/Internal/Infrastructure/HttpUtilities.Generated.cs#L41) and does some [pretty weird things](https://github.com/aspnet/KestrelHttpServer/blob/5bf7d3b28a2fcca56a7d24fd7e79c13dedae657b/src/Kestrel.Core/Internal/Infrastructure/HttpUtilities.Generated.cs#L65-L71) based on with the number/order

Reply:
Could just reset it to `Get` rather than `Unknown`; isn't a path where it can't get set to something